app/decorators/commands/decidim/participatory_processes/admin/update_participatory_process_decorator.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+# This decorator add suara permissions fields to update participatory process command
+Decidim::ParticipatoryProcesses::Admin::UpdateParticipatoryProcess.class_eval do
+  def attributes
```

Comment:
Can we use the same technique as in https://github.com/gencat/participa-catalunya/blob/master/app/decorators/decidim/courses/controllers/admin/courses_controller_decorator.rb to avoid the decorator being affected from changes in the command?